layouts/_default/baseof.html

```diff
@@ -19,15 +19,19 @@
 
 	{{ $style := resources.Get "css/style.css" | resources.ExecuteAsTemplate "css/style.css" . -}}
 	<link rel="stylesheet" href="{{ $style.RelPermalink }}">
-	{{ range .Site.Params.customCSS -}}
+	{{- range .Site.Params.customCSS }}
+	<link rel="stylesheet" href="{{ . | relURL }}">
+	{{- end }}
+	{{- range .Params.customCSS }}
 	<link rel="stylesheet" href="{{ . | relURL }}">
 	{{- end }}
 
 	{{- with .OutputFormats.Get "rss" }}
 	{{ printf `<link rel="%s" type="%s" href="%s" title="%s">` .Rel .MediaType.Type .RelPermalink $.Site.Title | safeHTML }}
 	{{- end }}
 
-	<link rel="shortcut icon" href="{{ "favicon.ico" | absURL }}">
+	{{ partial "favicon" . }}
```

Comment:
With the partial, it is possible to change just that "favicon" detail in the site proper, without having to copy the remainder of the file from the theme to the site.

Without the partial, an update to the theme would cause ripple on the site as well, i.e. one would have to re-do the copy of the redundant lines. Even though we're not really interested in them. Theme updates will be more stable if only a minimal set of changes needs to be on the using site, ideally no changes.